tests/wicked/advanced/sut/t19_team_interface_legacy.pm

```diff
@@ -0,0 +1,46 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Create a team interface from legacy ifcfg files
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+#             Jose Lausuch <jalausuch@suse.com>
+#             Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+use base 'wickedbase';
+use strict;
+use warnings;
+use testapi;
+use network_utils 'ifc_exists';
+use utils 'file_content_replace';
+
+
+sub run {
+    my ($self, $ctx) = @_;
+
+    my $cfg_team0 = '/etc/sysconfig/network/ifcfg-team0';
+    my $cfg_ifc0  = '/etc/sysconfig/network/ifcfg-' . $ctx->iface();
+    my $cfg_ifc1  = '/etc/sysconfig/network/ifcfg-' . $ctx->iface2();
+
+    record_info('Info', 'Create a team interface from legacy ifcfg files');
+
+    $self->get_from_data('wicked/teaming/ifcfg-eth0',             $cfg_ifc0);
```

Comment:
so you drop `ifcfg-eth0` file but you haven't replace it with something else here